### DISCUSSION TITLE
Question about new adblock

### DISCUSSION THREAD

**Discussion by devintly** [SpotX-Official/SpotX#133] [Question]
### Description of the new feature / enhancement

Hello, now we don't need chrome_elf.bak? And all works fine? But still can get old method throught -bts? Thank you in advance!

**Comment by amd64fox** [comment]
That's right, due to a recent glitch in bts, I decided to block all ads through spotx, but bts has already been fixed, now you can use it as before through the `-bts` incoming parameter in the installer.

At the moment, this is still experimental testing, if something changes, I will let everyone know.

**Reply by devintly** [reply]
Thank you, so it blocks ads in xpui or binares?) I don't undersrtand a little, but it is cool, that ads can be blocked without replacing chrome_elf)

**Reply by amd64fox** [reply]
spotx blocks all ads via code change in `xpui.js`

**Reply by Neurotoxin001** [reply]
Then why is chrome_self.zip still in the code?

**Reply by amd64fox** [reply]
Previously, spotx used bts as the main audio ad blocker, now spotx can do it itself and bts has been disabled by default, but anyone can use it in conjunction with spotx, just specify the `-bts` input parameter.

At the moment the bts patch is doing its job well, I left it as a backup method in case of emergency, as soon as it stops working I will update the spotx code.
